instructions/1-build-and-customize.md

```diff
@@ -155,52 +155,36 @@ walkthrough for setting it up</a>.
+    <img src="https://m.media-amazon.com/images/G/01/mobile-apps/dex/alexa/alexa-skills-kit/tutorials/general/2-9-lambda-function-role._TTH_.png" />
 
-7.  **Set up your Lambda function role.**  If you haven't done this before, we 
-have a <a href="https://github.com/alexa/alexa-cookbook/tree/master/aws/lambda-role.md" 
-target="_blank">detailed walkthrough for setting up your first role for Lambda</a>.  
-If you have done this before, set your **Existing role** value to 
-"lambda\_basic\_execution."
+8. **Click Create Function in the bottom right corner.**  You will need to scroll down to find **Create Function.**
 
-8. **Click the Create Function Button.**  
+    <img src="https://m.media-amazon.com/images/G/01/mobile-apps/dex/alexa/alexa-skills-kit/tutorials/general/2-11-create-function-button._TTH_.png" />
 
-9.  **Copy and paste the <a href="../lambda/custom/index.js" 
-target="_blank">provided code</a> into the Lambda function code box.**  We have 
-provided the code for this skill on <a href="../lambda/custom/index.js" 
-target="_blank">GitHub</a>. Delete the contents of the code box, and paste the 
-contents of the new code.
+9. **After you create the function, the ARN value appears in the top right corner. Copy this value for use in the next section of the guide.**
 
-10. **Click Save.**  
+    <img src="https://m.media-amazon.com/images/G/01/mobile-apps/dex/alexa/alexa-skills-kit/tutorials/quiz-game/2-12-copy-ARN._TTH_.png" />  <!--TODO: THIS IMAGE NEEDS TO BE CUSTOMIZED FOR YOUR SKILL TEMPLATE. -->
 
-11. **Click on Triggers** and click **+ Add Trigger**  
+10. **Configure your trigger.** Look at the column on the left called "Add triggers", and select Alexa Skills Kit from the list.  If you don't see Alexa Skills Kit in the list, jump back to step #3 on this page.
 
-12.  **Configure your trigger.** Click in the dashed box, and select Alexa Skills 
-Kit from the list.  If you don't see Alexa Skills Kit in the list, jump back to 
-step #3 on this page.   
-    <img src="https://m.media-amazon.com/images/G/01/mobile-apps/dex/alexa/alexa-skills-kit/tutorials/general/2-6-configure-your-trigger._TTH_.png" />  
-    Once you have selected Alexa Skills Kit, click the **Submit** button.    
+Once you have selected Alexa Skills Kit, scroll down and click the **Add** button. Then click the **Save** button in the top right. You should see a green success message at the top of your screen. Now, click the box that has the Lambda icon followed by the name of your function (HighLowGame if you used our suggestion) and scroll down to the field called "Function code".
 
-13. **After you create the function, the ARN value appears in the top right 
+7.  **Copy and paste the [provided code](../lambda/custom/index.js) into the Lambda function code box.**  We have provided the code for this skill on [GitHub](../lambda/custom/index.js).  Delete the contents of the code box, and paste the contents of the new code.
```

Comment:
Numbering has come out of sync. It shows 7 but should be 11

Reply:
Ah, md preview corrects those- fix coming for this and the following number.